pipeline/src/collection.py

```diff
@@ -85,6 +85,12 @@ def save(self, path, individual_files=False, include_empty_properties=False):
         # we first re-add all child nodes to the collection.
         # This is probably not the most elegant or fast way to do this, but it is simple and robust.
         for node in tuple(self.nodes.values()):
+            
+            if node.type_.startswith("https://openminds.ebrains.eu/"):
```

Comment:
This is not the best way to detect the version. Perhaps we should add a property to the collection JSON file when saving, indicating which version of openMINDS it was produced with.